lego/apps/surveys/views.py

```diff
@@ -52,6 +55,8 @@ class SubmissionViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
     filter_class = SubmissionFilterSet
     filter_backends = (DjangoFilterBackend, )
 
+    # authentication_classes = api_settings.DEFAULT_AUTHENTICATION_CLASSES + [SurveyAuthentication]
```

Comment:
Don't I need this here as well? Seems to work without, but I don't understand how